openlcs/tasks/views.py

```diff
@@ -117,24 +119,26 @@ def list(self, request, *args, **kwargs):
 
             [
                 {
-                    "id": 20,
-                    "meta_id": "82b02650-5d36-4430-876a-c8401266b438",
+                    "id": 5,
+                    "meta_id": "60747ba3-653f-4669-b37e-08eb8b3ae14f",
                     "owner": "admin",
-                    "params": "{\"package_nvr\": \"fio-3.1-2.el7\", \"license_scan\": true,
-                    \"copyright_scan\": true, \"product_release\": \"satellite-6.9.0\"}",
+                    "params": "{\"package_nvr\": \"ansible-2.4.2.0-2.el7\", \"license_scan\": true,
+                     \"copyright_scan\": true, \"product_release\": \"satellite-6.9.0\"}",
                     "status": "SUCCESS",
-                    "date_done": "2022-10-24T09:17:07.406133",
-                    "traceback": null
+                    "date_done": "2022-11-10T05:39:45.746881",
+                    "traceback": null,
+                    "object_url": "http://10.0.213.0:8000/rest/v1/sources/2/"
                 },
                 {
-                    "id": 19,
-                    "meta_id": "3808b332-45df-4fa2-a4ca-3225549adc2a",
+                    "id": 6,
+                    "meta_id": "ddc8d40a-3c85-4a14-ace8-440c7e69ad55",
                     "owner": "admin",
-                    "params": "{\"package_nvr\": \"ansible-2.4.2.0-2.el7\", \"license_scan\": true,
+                    "params": "{\"package_nvr\": \"fio-3.1-2.el7\", \"license_scan\": true,
                      \"copyright_scan\": true, \"product_release\": \"satellite-6.9.0\"}",
-                    "status": "STARTED",
-                    "date_done": null,
-                    "traceback": null
+                    "status": "SUCCESS",
+                    "date_done": "2022-11-10T05:38:50.461372",
+                    "traceback": null,
+                    "object_url": "http://10.0.213.0:8000/rest/v1/sources/1/"
```

Comment:
you may need to replace it with `settings.HOSTNAME` instead of hard code. Also curious whether it's better to use `source_url`? it's fine to keep it as it is too.

Reply:
This is a doc, in real code where generate the url, i use settings.HOSTNAME and django.urls.reverse function to generate it.You mean in doc code, i should also use the settings.HOSTNAME.
The name object_url is duanmu recommended.She says there may be other types in future, what is your suggestions.

Reply:
I already said in original comment. also said It's fine to keep it.

Reply:
Sorry, missed that part